docs/guides/other-guides/n8n.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Using ngrok with n8n to make your workflow public 
+description: Learn how to use ngrok to make your n8n workflow accessible from anywhere, not just your local host. 
+---
+
+import TabItem from "@theme/TabItem";
+import Tabs from "@theme/Tabs";
+
+[n8n allows you to self-host](https://docs.n8n.io/hosting/?) which is great when you're testing workflows, but when you're ready to turn your instance into a production-ready automation and don't want to buy a domain or pay for hosting, you might feel lost. This tutorial walks you through how to get your self-hosted n8n instance running in Docker live on the internet. We'll show you how to set everything up in Docker, get your free static domain, and how to protect your endpoint, all for free with ngrok.
+
+## Set up your local server with n8n
+
+Follow the steps listed in [n8n's docs](https://docs.n8n.io/hosting/installation/docker/) to self-host n8n in Docker.
+
+### 1. Install ngrok
+
+[Download and install](https://ngrok.com/download) the ngrok CLI.
+
+**For Docker**
+Install ngrok via Docker with the following command:
+
+```
+docker pull ngrok/ngrok
+```
+
+Run ngrok via Docker:
+
+```
+docker run -it -e NGROK_AUTHTOKEN=<token> ngrok/ngrok http 80
```

Comment:
Flow here seems out of order. This mentions the auth token but next step is obtaining it.